services/movie.py

```diff
@@ -1,18 +1,23 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+        title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
-    if genres_ids:
+    if title is not None:
+        queryset = queryset.filter(title__contains=title)
+
+    if genres_ids is not None:
```

Comment:
if genres_ids is perfectly fine here :)

Reply:
> if genres_ids is perfectly fine here :)

what if ```genres_ids = []``` instead of ```None```, we have to lookup at it to check what we ment, I belive it gives more readablility when we specify ```is not None```

Reply:
empty list evaluates as false no?

Reply:
> empty list evaluates as false no?

probably right here, I don't remember the case but I once asked GPT about checking for None if value = None and he said that condition ```is not None``` were prefereable there.

Reply:
Well GPT changes his mind pretty easy :)
`is not None` assumes we are checking only if param is actually None but other falsy values may get through